lib/request.js

```diff
@@ -98,7 +98,7 @@ internals.Request.prototype.prepare = function (next) {
 
     const chunks = [];
 
-    this._shot.payload.on('data', (chunk) => chunks.push(chunk));
+    this._shot.payload.on('data', (chunk) => chunks.push(typeof chunk === 'string' ? Buffer.from(chunk) : chunk));
```

Comment:
This could just be `chunks.push(Buffer.from(chunk))` but this will break with node < 4.5.0. Checking for a string gives better backwards compatibility.